distribution/docker/build.gradle

```diff
@@ -165,6 +165,43 @@ tasks.register("copyKeystore", Sync) {
   }
 }
 
+tasks.register("checkSecurityAuditLayoutPatternIdentical") {
+  // the two log4j2.properties files containing security audit configuration for archive and docker builds respectively
+  def originalLog4j = project(":x-pack:plugin:core").file('src/main/config/log4j2.properties')
+  def dockerLog4j = project.projectDir.toPath().resolve("src/docker/config/log4j2.properties").toFile()
```

Comment:
you can just do `project.file("src/docker/config/log4j2.properties")`

Reply:
Thanks 🤦